README.md

```diff
@@ -116,7 +148,7 @@ This directory will contain the following files:
 - word_results.csv The per-word probability by topic scores. Each line represent the conditional probabilities of a word. First entry is the word (a summarized network event), and this is separated from the remainder by a comma. The remainder of the line is a space-separated list of floating-point numbers. The number at position k is the probability of seeing this word conditioned on being in topic k.
 - doc.dat An intermediate file mapping integers to IP addresses.
```

Comment:
New architecture is not persisting doc.dat or words.dat information. It's all in memory until mode.dat creation.
Please let me know if we should be persisting this data.